### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by DeprecatedNether**
I'm sure pempkin pie can be tasty but I'd prefer pumpkin pie :+1:

### CODE REVIEW

File: Weapon.php [hunk 1]
```diff
@@ -83,7 +83,7 @@ class Weapon
         'ROTTEN_FLESH' => 'Pike', 'POTATO' => 'Halberd', 'MELON' => 'Divine Reach',
         'POISONOUS_POTATO' => 'Ruby Thorn', 'STRING' => 'Hammer of Light', 'RAW_CHICKEN' => 'Nethersteel Katana',
         'MUTTON' => 'Claws', 'PORK' => 'Mandibles', 'RAW_BEEF' => 'Katar',
-        'APPLE' => 'Enderfist', 'PEMPKIN_PIE' => 'Orc Axe', 'COOKED_COD' => 'Doubleaxe',
```
Comment (Rapidd) [inline Weapon.php]: Was it delicious?

**Comment by MaxKorlaar** [comment]
:+1:

File: Weapon.php (hunk 1 above)
Comment (Rapidd) [inline Weapon.php]: +You can't blame castro for this

**Comment by Rapidd** [comment]
Would you like cream on that pempkin pie?

**Pull request merged**